Library/Formula/cctools.rb

```diff
@@ -11,6 +11,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation
     cellar :any
```

Comment:
You also need to update [this line](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/bottle.rb#L197) to make an exception for `cctools`.